source/vibe/textfilter/urlencode.d

```diff
@@ -40,12 +42,12 @@ string urlEncode(string str, string allowed_chars = null)
 	return str;
 }
 
-unittest {
+@safe unittest {
 	string s = "hello-world";
 	assert(s.urlEncode().ptr == s.ptr);
 }
 
-private bool isCorrectHexNum(string str)
+private auto isCorrectHexNum(const(char)[] str)
```

Comment:
It's present since 2.068: http://dlang.org/changelog/2.068.0.html#attribinference3